amass/sources/sources_test.go

```diff
@@ -94,3 +104,31 @@ loop:
 
 	return count
 }
+
+func outputDirectory(dir string) string {
+	if dir == "" {
+		if path, err := homedir.Dir(); err == nil {
+			dir = filepath.Join(path, handlers.DefaultGraphDBDirectory)
+		}
+	}
+	return dir
+}
+
+func acquireConfig(dir, file string, config *core.Config) (string, bool) {
```

Comment:
I think it makes more sense to move this function (and outputDirectory) into the `amass/core` package. Then the tests and the commandline applications can leverage the same code without having to maintain separate copies.

Reply:
I agree. The same function exists in the code specific to the tool and can be moved after this PR is merged.

Reply:
I also agree. Working on it.

Reply:
Done